plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverApplication.java

```diff
@@ -264,6 +249,21 @@ public Object start(IApplicationContext context) {
 
         initializeApplication();
 
+        // Custom parameters
```

Comment:
Wrong. We cannot process custom parameters after we initialized UI.
It will break any headless tasks (e.g. data export task which mustn't trigger UI and must work in pure headless mode)

Reply:
The method extracted and passes to a regular initialization application circle.